packages/t/typescript-vfs.json

```
@@ -0,0 +1,32 @@
+{
+  "description": "A Map based TypeScript Virtual File System.",
+  "filename": "dist/vfs.globals.js",
+  "homepage": "http://www.typescriptlang.org",
+  "keywords": [
+    "typescript",
+    "vfs"
+  ],
+  "license": "MIT",
+  "name": "typescript",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/TypeScript-Website"
+  },
+  "autoupdate": {
+    "source": "npm",
+    "target": "@typescript/vfs",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
```

```suggestion
          "*.js"
```
We try to avoid adding globstars, we'd prefer a glob that is more specific to the actual files that're needed.